src/admin/admin.service.ts

```
@@ -1449,14 +1733,33 @@ export class AdminService {
+      'Nov',
+      'Dec',
+    ];
 
     // Initialize map
-    const dataMap = new Map<string, { elevenLabs: number; gemini: number; total: number }>();
-    months.forEach(m => dataMap.set(m, { elevenLabs: 0, gemini: 0, total: 0 }));
+    const dataMap = new Map<
+      string,
+      { elevenLabs: number; gemini: number; total: number }
+    >();
+    months.forEach((m) =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)